<launch>
  <arg name="URDF_FILE" value="$(find msl_hand_controller)/models/JAXON_RED_SENSORS.urdf"/>
  <param name="robot_description" textfile="$(arg URDF_FILE)" />
  <param name="robot_model_file" value="$(arg URDF_FILE)" />
  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" />
</launch>
